src/Morphir/Spark/AST.elm

```diff
@@ -596,6 +639,35 @@ mapSDKFunctions ir args fQName =
                 |> Result.map List.singleton
                 |> Result.map (Function "sum")
 
+        -- HANDLE AGGREGATION FUNCTIONS
+        ( "Morphir.SDK:Aggregate:count", [] ) ->
+            -- morphir's count takes no arguments, but Spark's does
+            Function "count" [ Function "lit" [ Literal (WholeNumberLiteral 1) ] ] |> Ok
+
+        ( "Morphir.SDK:Aggregate:sumOf", item :: [] ) ->
+            expressionFromValue ir item
+                |> Result.map List.singleton
+                |> Result.map (Function "sum")
+
+        ( "Morphir.SDK:Aggregate:averageOf", item :: [] ) ->
+            expressionFromValue ir item
+                |> Result.map List.singleton
+                |> Result.map (Function "avg")
+
+        ( "Morphir.SDK:Aggregate:minimumOf", item :: [] ) ->
+            expressionFromValue ir item
+                |> Result.map List.singleton
+                |> Result.map (Function "min")
+
+        ( "Morphir.SDK:Aggregate:maximumOf", item :: [] ) ->
+            expressionFromValue ir item
+                |> Result.map List.singleton
+                |> Result.map (Function "max")
+
+        ( "Morphir.SDK:Aggregate:weightedAverageOf", _ ) ->
+            -- XXX: I can't see a Spark equivalent
```

Comment:
Can you please create an issue for this? I'm not sure if we will prioritize it but we should at least keep track.

Reply:
I have issue #845 to keep track of this